src/sentry/search/events/builder/discover.py

```diff
@@ -219,6 +223,12 @@ def __init__(
         # value_resolver_map may change type
         self.meta_resolver_map: Dict[str, str] = {}
 
+        self.skip_tag_resolution = skip_tag_resolution
+        # prefixed tag: tag map for post-processing values
+        self.tag_resolver_map: Dict[str, str] = {}
+        # tag: prefixed tag map for post-processing values
+        self.reverse_tag_resolver_map: Dict[str, str] = {}
```

Comment:
nit: we should name these to be more descriptive something like prefixed_to_tag_map and tag_to_prefixed_map or something along those lines

Reply:
Yep, these names are kind of confusing so I changed them to your suggestions. That should make it clearer. I also added a better comment that's more specific about what they're there for.